actions/next-semver/.nvmrc

```diff
@@ -0,0 +1 @@
+16.17.1
```

Comment:
Why not 18.12.1?

Reply:
16x is the latest available GHA runtime for node, see https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs-for-javascript-actions